### ISSUE TITLE
Form to sign up as research participant  

### ISSUE THREAD

**Issue by andycochran** [HHS/simpler-grants-gov#758]
### Summary

Design a form to collect necessary info from potential research participants for recruiting

- [x] Define required fields (What minimum data are we collecting?) 
- [x] Build out the intercept form in Ethnio
- [x] Design the UI/UX on Simpler.Grants.gov for getting to the form (from research page?)
- [x] Get PRA clearance (work w/ @itsemilyianacone on this)
    - 11/25/2024 per new guidance, PRA is not needed on form inviting people to participate in usability studies

#### Context

An earlier design iteration of the [Research](https://simpler.grants.gov/research) page included a call-to-action where _anyone_ who's interested could sign up to be a potential research participant ([Figma](https://www.figma.com/design/lpKPdyTyLJB5JArxhGjJnE/simpler.grants.gov?node-id=1412-606&node-type=frame&t=RLRQN5SVrc8NyDdG-0)): 
![Image](https://github.com/user-attachments/assets/0c7919aa-c9ec-4d7d-9fed-aa5eec6639a7)

This was deferred, as we had not yet resolved what the data collection method would be. We instead put in a newsletter signup link. We should revisit the design of this call-to-action now that Ethnio is set up to collect contact info of interested participants. 

#### Open questions

- Should this just be a link to an Ethnio form? Or is there an Ethnio API we should leverage to build a custom page for a more seamless UX? 
- What is the minimum amount of PII we need to collect? 
  - e.g. email? name? applicant or grantor? archetype? demographics? 
  - Can we infer from minimal data whether compensation is permissible? 
- What happens if someone signs up multiple times? 
- Can volunteers cancel/revoke their signup? 

### Acceptance criteria

- [ ] Design & content approved by HHS
- [ ] Simpler.Grants.gov has received PRA clearance to collect potential research participant info/data 
- [ ] Users from general public can sign up as interested in participating in future research 
- [ ] Contact info is stored securely in Ethnio

**Comment by sumiat** [comment]
Blocked until we can procure Ethnio cc @andycochran @EOKENAVA

**Comment by EOKENAVA** [comment]
Blocked until we can procure Ethnio cc @sarahknoppA6

**Comment by snyder-ux** [comment]
If participant signs up multiple times: we can manually link the accounts.
- Is there an automatic suggestion or easy view based on identical email address?
- If participant unsubscribes, is it based on the email address, regardless of the accounts being linked?

**Comment by snyder-ux** [comment]
draft in Ethnio, screener titled [Research Participant Sign Up](https://ethn.io/screeners/113856/edit?step=design) and [preview](https://ethn.io/preview/91822#js)

**Comment by andycochran** [comment]
Let's review/critique this in tomorrow's Design Sync?

**Comment by andycochran** [comment]
@michellemin-nava, please work with @snyder-ux to review the content for this. A few questions I have looking at the preview: 
- We want to make it clear that folx are not signing up for a particular study. They may never be selected. 
  - Maybe consider "volunteer" vs "sign up" and avoid "we will" (since we might not). 
  - The confirmation message after submission also implies we _will_ send them an invite
- Should we include the word "studies"? 
- "Role" might be confusing. Should we ask a more direct question about how they use grants.gov and list the options as "I…" statements? 
- A novice applicant might not consider themselves an applicant if they've never actually applied. That option could include both searching and/or applying. 
- I don't think we do, but confirm we don't need _full_ name

**Comment by snyder-ux** [comment]
- **PRA** [0990-0379 MEMO_FastTrack Generic Clearance Submission](https://docs.google.com/document/d/1N_3-U39LVaZbz6p0hhwC4BgWx2gHy5hcHDQ83XYjmog/edit?usp=drive_link)
- [0990-0491](https://drive.google.com/file/d/1JQ541lmKuIf5AMZqslAs14H7IsNj5UkG/view?usp=sharing)

**Comment by snyder-ux** [comment]
11/5 PRA update from Andy C in Slack:
yesterday Emily Ianacone let us know in Design Review that she had to resubmit the PRA requests under the proper codes. There’s a new general clearance we’re using. I think this reset the clock. Again. So hopefully all our PRA requests get approved this week.

**Comment by snyder-ux** [comment]
11/18: PRA has technical issue on OMB’s side and waiting on instructions on how to transfer the files to get them under review, Emily will continue to keep us updated.

**Comment by snyder-ux** [comment]
11/25/2024
-  Mtg with Emily L and confirmed per new guidance, PRA is not needed on form inviting people to participate in usability studies
-  Made minor edit to form to use same language as guidance (changed “research study” to “usability study” to match guidance). Emily approved form ([Ethn.io preview](https://ethn.io/preview/91822#js))
- @michellemin-nava provided copy edits to the text on the page
- [Figma](https://www.figma.com/design/FGxWtAgToKhehLJCiuy1zL/Simpler.Grants.gov-Project?node-id=8-28176&t=fjryKyS5Xr85lno0-1)

**Comment by andycochran** [comment]
Is there an engineering ticket to implement this? 
cc @mdragon

**Comment by mdragon** [comment]
@andycochran I think it's #773 which ironically Doug just shipped.
